modules/light-clients/08-wasm/types/client_state.go

```diff
@@ -47,7 +47,7 @@ func (cs ClientState) Validate() error {
 	if lenCodeHash == 0 {
 		return errorsmod.Wrap(ErrInvalidCodeHash, "code hash cannot be empty")
 	}
-	if lenCodeHash > 32 { // sha256 output is 256 bits long
+	if lenCodeHash != 32 { // sha256 output is 256 bits long
```

Comment:
only nit pertains to err message used here, doesn't really give indication that `32` refers to # of bytes.